packages/server/src/api/rest/index.ts

```diff
@@ -1178,18 +1182,31 @@ class RequestHandler extends APIHandlerBase {
         return r.toString();
     }
 
-    private makeIdFilter(idField: string, idFieldType: string, resourceId: string) {
-        return { [idField]: this.coerce(idFieldType, resourceId) };
+    private makeIdFilter(idFields: IdField[], resourceId: string) {
+        if (idFields.length === 1) {
+            return { [idFields[0].name]: this.coerce(idFields[0].type, resourceId) };
+        } else {
+            return {
+                [idFields.map((idf) => idf.name).join('_')]: idFields.reduce(
+                    (acc, curr, idx) => ({
+                        ...acc,
+                        [curr.name]: this.coerce(curr.type, resourceId.split(idDivider)[idx]),
```

Comment:
This means the id values cannot contain the divider.

I like the fact that if we join id fields with "_", the URL will be easy to read and understand. A more reliable approach is to keep it as an object and URL encode it, but it'll look quite ugly.

I'm a bit uncertain about the tradeoff ... what do you think 😄?

Reply:
I had the same thought. I ended up on this setup for two reasons. First it matches the Prisma default, secondly it works without changing the URLPattern config, since `_` is in the [default list of accepted characters](https://github.com/snd/url-pattern?tab=readme-ov-file#named-segments). 

If it turns out to be problematic in practice this may need to be changed. Either like you say an URL encoded object, or maybe making the id divider configurable.

Reply:
I don't have a strong opinion about it. I like keeping the URL clean and readable. Maybe as you said making the id divider configurable (in the future) is a good solution. Let's keep it as is for now.